### ISSUE TITLE
Select options not visible in "alternate" box

### ISSUE THREAD

**Issue by LukasChalupa** [contember/admin#348]
Happens with any select (SelectField, MultiSelectField) inside alternate box

Block editor example to replicate:
```
<Block
	discriminateBy="asideContent"
	label="Aside content"
	alternate={
		<SelectField
			field="alignment"
			label="Alignment"
			options={[
				{ value: 'left', label: 'Text left' },
				{ value: 'right', label: 'Text right' },
			]}
			defaultValue="left"
		/>
	}
>
	Block content
</Block>
```

![image](https://user-images.githubusercontent.com/5323575/172835896-63fa156e-6b8c-41b6-ab35-215694f0cf74.png)

**Comment by honzasladek** [comment]
@attitude 1.0.3

**Comment by attitude** [comment]
Seems to be already fixed in #338. I can confirm it working in the upcoming v1.1 branch, but wasn't back-ported. I will cherry pick the commit for the v1.0 branch.

<img width="762" alt="Contember Admin – fixed dropdown overflows" src="https://user-images.githubusercontent.com/387611/173048080-e7560f89-007e-4b42-bf80-ce9d5acccf26.png">

Cc: @honzasladek & @LukasChalupa

<img width="762" alt="Screenshot 2022-06-10 at 12 32 08" src="https://user-images.githubusercontent.com/387611/173047301-0a044190-752b-4475-8d29-4236531202bd.png">
